test/unit/lodash.spec.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+var chai = require('chai');
+var expect = chai.expect;
+var _ = require('lodash');
+var datasets = require('../../app/datasets');
+var Guy = require('./Guy');
+var chance = require('chance').Chance();
+
+describe('Lodash training', function ()
+{
+    describe('Number functions', function () {
+        describe('clamp', function () {
+
+            var params, number, lower, upper;
+
+            before(function () {
+                params = datasets.clamp();
+                number = params[0];
+                lower = params[1];
+                upper = params[2];
+            });
+            
+            it('should verify params', function () {
+                expect(params).to.have.length(3);
+                expect(lower < upper).to.eql(true);
```

Comment:
http://chaijs.com/api/bdd/#method_below below method will fit better in this case.